src/main/res/values/strings.xml

```diff
@@ -344,5 +344,6 @@
     <string name="size_largest">Size (largest first)</string>
     <string name="size_smallest">Size (smallest first)</string>
     <string name="file_type">File type</string>
+    <string name="message_toast_press_back_again">Press back again to exit</string>
```

Comment:
Please add translations in other languages.

Reply:
Done

Reply:
Not something related to this PR, but for a few days, I am not able to login to the Amahi app. It's showing a connection error. Earlier it was working fine.

Reply:
Please approve changes

Reply:
@chirag-jn 
I have done the required changes already. Please approve the PR.